### DISCUSSION TITLE
ModuleNotFoundError: No module named 'mxnet'

### DISCUSSION THREAD

**Discussion by mahmoodn** [apache/incubator-mxnet#14059] [Show and tell] [answered]
Hi,
Although I have built mxnet with `make -j8 USE_OPENCV=1 USE_BLAS=openblas USE_CUDA=1 USE_CUDA_PATH=/usr/local/cuda USE_CUDNN=1`, I am not able to run an example

$ python main.py --configfile default.cfg
Traceback (most recent call last):
  File "main.py", line 23, in <module>
    from config_util import parse_args, parse_contexts, generate_file_path
  File "/home/mahmood/mx/mxnet/example/speech_recognition/config_util.py", line 23, in <module>
    import mxnet as mx
ModuleNotFoundError: No module named 'mxnet'

I think I missed some steps. As stated [here](https://mxnet.incubator.apache.org/versions/master/install/build_from_source.html), there a Ninja thing! Is that mandatory?

[a comment by an automated account is not shown]

**Comment by andrewfayres** [comment]
@mxnet-label-bot add [installation]

It's generally recommended to use pip to install MXNet unless you've got a need to build from source. You can find instructions [here](https://mxnet.incubator.apache.org/versions/master/install/index.html?platform=Linux&language=Python&processor=GPU).

If you're still struggling you might try posting at discuss.mxnet.io for help. That tends to be a better place for getting help and asking questions as the github is primarily for tracking issues, and feature requests.

**Comment by akashAD98** [comment, marked as answer]
you can use this command `pip install mxnet`
